spec/fixtures/cli/theme-ls

```diff
@@ -1,4 +1,3 @@
-tmp/theme-test:
 public
```

Comment:
Why the change in this file? Seems to also be related to why the CI tests fail?

Reply:
Sorry. I don't know why the output of `ls -R #{theme_dir}` is different on my system. I'm going to recover the previous content and comment that test in local to be able to check the other ones.

Reply:
No worries. I will change the test to be more tolerant after we merge this.